docs/backends/quickwit.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "Quickwit"
+---
+
+## Configuring Quickwit Backend
+
+The only required field is the Quickwit endpoint which is supporting OTLP grpc (it remain the same http endpoint used for the Quickwit RESTful API).
+This endpoint must be a valid URL for your Quickwit cluster, i.e: `http://{quickwit_host}:7281`
```

Comment:
For other destinations, if the port is not specified, odigos will use the default port for this backend. I wonder if quickwit can also benefit from this behavior.

You can use or take inspiration from [this function](https://github.com/keyval-dev/odigos/blob/802b3322f17a092d48e4307761f0ea7cffc8519f/autoscaler/controllers/gateway/config/utils.go#L10)